README.md

```diff
@@ -30,21 +33,12 @@ Something bad is happening in Israel, and we cannot afford to stand still. We've
   Inside the project directory, `cp .env .env.local` and fill the `REACT_APP_FIREBASE_*` variables with the firebase configuration being shown.
 - Go back to firebase dashboard and select **Cloud Firestore** from the sidebar. Click **Create Database**, start in test mode, and press enable.
 
-### Google Maps API key
-
-We are using Google's Places Autocomplete & Geolocation APIs on the `PlacesAutocomplete` component. You need to [get a Maps API Key](https://developers.google.com/maps/documentation/javascript/get-api-key) for them to work.
-
-After getting one, add it to `REACT_APP_GOOGLE_MAPS_API_KEY` inside `.env.local` file.
-
-> Note: Please help us [find an alternative API provider](https://github.com/guytepper/1km.co.il/issues/6) for those services.
-
 ## Add protests
 
 Adding protests for development is be a bit cumbersome currently. We'll improve the flow as we go.
 
-- First, remove all things related to Recaptcha. It's in the [API file](https://github.com/guytepper/1km.co.il/blob/master/src/api/index.js#L30-L32) and in the `ProtestForm` component (in the form component search for all references to `recaptcha` and make sure they don't get along in the flow).
-- Change `pending_protests` collection to `protests` [in the createProtest function](https://github.com/guytepper/1km.co.il/blob/master/src/api/index.js#L34).
-- Add a protest from `/add-admin` 🤗
+- Change `pending_protests` collection to `protests` [in the createPendingProtest API mehtod](https://github.com/guytepper/1km.co.il/blob/master/src/api/index.js#L33).
```

Comment:
How about making it controlled by `process.env.NODE_ENV`?

Reply:
Genius <3